dhd.csl

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
+  <info>
+    <title>DHd (Deutsch)</title>
+    <title-short>DHd (Deutsch)</title-short>
+    <id>http://www.zotero.org/styles/dhd</id>
```

Comment:
Please spell out DHd in full in the title and ID. Please rename the CSL file accordingly

Reply:
Done…